community/planning/roadmap.md

```diff
@@ -21,17 +21,17 @@ of downtime for upgrades or single-point-of-failure issues.
 
 | Feature | Status | Documentation |
 | ------- | ------ | ------------- |
-| Circuit Deletion | Not Started | - |
-| Circuit Name | Under Development | - |
+| Circuit Deletion | Under Development | - |
+| Circuit Name | Complete | [splinter circuit propose]({% link docs/0.5/references/cli/splinter-circuit-propose.1.md %}), [splinter circuit show]({% link docs/0.5/references/cli/splinter-circuit-show.1.md %}), [splinter circuit list]({% link docs/0.5/references/cli/splinter-circuit-list.1.md %}) |
 | Circuit Template | Complete | [How-to]({% link docs/0.5/howto/using_circuit_templates.md %}), [Man page]({% link docs/0.5/references/cli/splinter-circuit-propose.1.md %}) |
 | Challenge Authorization | Not Started | - |
-| Cylinder Support | Under Development | [Repo](https://github.com/Cargill/cylinder) |
+| Cylinder Support | Complete | [Repo](https://github.com/Cargill/cylinder) |
 | Cloud-friendly Deployment | Under Development | [Admin Service Store Feature]({% link community/planning/admin_service_store.md %}), [Data Store Guidelines]({% link community/data_store_guidelines.md %}) |
-| Database - PostgreSQL Support | Under Development | - |
-| Database - SQLite Support | Under Development | - |
-| REST API - Authorization | Discussion | - |
-| REST API - Cylinder JWT Authentication | Under Development | - |
-| REST API - OAuth 2 Authentication | Under Development | - |
+| Database - PostgreSQL Support | Under Development | [Data Store Guidelines]({% link community/data_store_guidelines.md %}) |
+| Database - SQLite Support | Complete | [Data Store Guidelines]({% link community/data_store_guidelines.md %}) |
```

Comment:
This is not complete yet, at least two features still need to be stabilized

Reply:
Corrected